### PR TITLE
account reset overlay storybook

### PR DESCRIPTION
**Pull request by pzduniak**
Text is a placeholder pending DESIGN-2616, icon is pending #18822 (though this PR should go first). The idea is that this will be slightly customized and all the plumbing will be done in Y2K-160.

@keybase/react-hackers 
@keybase/y2ksquad

### CODE REVIEW

File: shared/login/reset-modal/container.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as Container from '../../util/container'
```
Comment (buoyad) [inline shared/login/reset-modal/container.tsx]: IMO let's delete this until it's integrated

Comment (pzduniak) [reply shared/login/reset-modal/container.tsx]: it's there to make it testable, otherwise you pretty much cant see the whole thing in action

Comment (pzduniak) [reply shared/login/reset-modal/container.tsx]: BUT this whole thing might be changed, there's been changes in what we want to put in here

Comment (buoyad) [reply shared/login/reset-modal/container.tsx]: It's usually the integration's job to add the container and in-app entry point for this at once

**Pull request merged**